### ISSUE TITLE
Combine Admin Options and Custom Options with same field id

### ISSUE THREAD

**Issue by orhaneroglu001** [Codestar/codestar-framework#29]
Hello,
The fields  do not display in the Customize page.

Despite the addition of  " 'show_in_customizer' => true"

```
// Control core classes for avoid errors
if( class_exists( "CSF" ) ) {

  //
  // Set a unique slug-like ID
  $prefix = "my_framework";

  //
  // Create options
  CSF::createOptions( $prefix, array(
    'menu_title' => 'My Framework',
    'menu_slug'  => 'my-framework',
    'show_in_customizer' => true,
  ) );

  //
  // Create a section
  CSF::createSection( $prefix, array(
    'title'  => 'Tab Title 1',
    'fields' => array(

      //
      // A text field
      array(
        'id'    => 'opt-text',
        'type'  => 'text',
        'title' => 'Simple Text',
      ),

    )
  ) );

  //
  // Create a section
  CSF::createSection( $prefix, array(
    'title'  => 'Tab Title 2',
    'fields' => array(
      

      // A textarea field
      array(
        'id'    => 'opt-textarea',
        'type'  => 'textarea',
        'title' => 'Simple Textarea',
        
      ),

    )
  ) );

}
```
Where is the problem ?
Thank you :)

**Comment by narotemporary** [comment]
After having searched the documentation I think I found the solution but the fields still do not display in the Customize page anyway.

I add " 'show_in_customizer' => true"
update my code :

```
// Control core classes for avoid errors
if( class_exists( "CSF" ) ) {

  //
  // Set a unique slug-like ID
  $prefix = "my_framework";

  //
  // Create options
  CSF::createOptions( $prefix, array(
    'menu_title' => 'My Framework',
    'menu_slug'  => 'my-framework',
    'show_in_customizer' => true,
  ) );

  //
  // Create a section
  CSF::createSection( $prefix, array(
    'title'  => 'Tab Title 1',
    'fields' => array(

      //
      // A text field
      array(
        'id'    => 'opt-text',
        'type'  => 'text',
        'title' => 'Simple Text',
      ),

    )
  ) );

  //
  // Create a section
  CSF::createSection( $prefix, array(
    'title'  => 'Tab Title 2',
    'fields' => array(
      

      // A textarea field
      array(
        'id'    => 'opt-textarea',
        'type'  => 'textarea',
        'title' => 'Simple Textarea',
        
      ),

    )
  ) );

}
```

**Comment by Codestar** [comment]
Hi,

Yes you're right and it fixed in v2.0.9 can you try it ? :)

Regards, Codestar
